test/integration/connect/envoy/case-expose-checks/service_s2.hcl

```diff
@@ -0,0 +1,22 @@
+services {
+  name = "s2"
+  port = 8181
+  connect {
+    sidecar_service {
+      proxy {
+        expose {
+          checks = true
+        }
+      }
+    }
+  }
+  checks = [
+    {
+      name     = "http"
+      http     = "http://127.0.0.1:8181/debug"
```

Comment:
The fake workloads are usually copies of `fortio` which has a webserver builtin. One of the pages is `/debug` so that'll 200, hence why it's used here as the healthcheck standin.